lib/ts/utils.ts

```diff
@@ -675,7 +676,7 @@ export async function runProjectOrPrintStartCommand(answers: Answers, userArgume
         return;
     }
 
-    let appRunScript = `${await getPackageManagerCommand(userArguments)} run start`;
+    let appRunScript = `${userArguments.manager} run start`;
 
     if (selectedFrontend.isFullStack) {
```

Comment:
also need a version change:
- version.ts
- package.json

Reply:
🫡 May I add this to the contribution guide to always update the version?

Reply:
It's there in the PR checklist template above already. But yea, adding to the contributing guide always helps too :)

Reply:
Aaand done!